usage/jsgui/src/main/webapp/assets/js/config.js

```diff
@@ -48,6 +50,15 @@ require.config({
         "zeroclipboard":"libs/ZeroClipboard",
         "js-yaml":"libs/js-yaml",
         
+        "codemirror":"lib/codemirror",
```

Comment:
this is true. This is the reason I put ```addon```, ```mode``` in the lib, and not in the codemirror folder. I tried modifying the ```config.js``` as explained in doc but it didn't work. 

Let me know if it works for you ...